dime-research-standards/pillar-4-data-security/data-security-resources/veracrypt-guidelines.md

```diff
@@ -4,7 +4,43 @@
 
 VeraCrypt is an open source software that is free of charge. While untested open source software may be risky to use, VeraCrypt is well-established open source software that has been reviewed by a large numbers of cybersecurity experts who use it themselves. It is widely accepted as your safest option.
 
-## Guidelines for each part of a VeraCrypt workflow
+## Why VeraCrypt and not alternatives like BoxCryptor
+ 
+DIME Analytics cannot support all software. 
+We select the software that we think is best and 
+can be used in the greatest number of contexts 
+so as few software as possible needs to be learned by the project teams. 
+This does not mean that the software we do not select are all terrible. 
+We selected VeraCrypt over competitors as BoxCryptor as we think VeraCrypt is 
+more secure, can be used in more context and will always be free to use.
+
+Here are the reasons for why we think VeraCrypt is the better alternative:
+ 
+* BoxCryptor can only be used when securing content in a synced folder in for example DropBox. 
+VeraCrypt can also be used for that in addition to other use cases, 
+such as encrypting files to be sent over email or encrypting files stored on an external hard drive. 
+If you learn VeraCrypt you do not need to learn other encryptions software
+for any other use case common in a typical research project. 
+* VeraCrypt is open source and thousands of independent security experts 
+have scrutinized the implementation instead of only a handful who are paid by BoxCryptor.
```

Comment:
```suggestion
* VeraCrypt is open-source has been verified by independent security experts, 
rather than being a proprietary and closed software.
Such tools have had serious security flaws in the past and cannot be independently verified.
```